src/components/Sidebar.vue

```diff
@@ -90,7 +91,7 @@ export default {
   }
 
   .sidebar-datepicker {
-    position: absolute;
+    //position: absolute;
```

Comment:
Commented-out code can be omitted entirely as we have version control to revert back.